### ISSUE TITLE
Seria needs loads and dumps

### ISSUE THREAD

**Issue by theherk** [rtluckie/seria#14]
Am I correct that it currently requires a file object? It would be nice to take a reply from a server in xml and do `seria.loads(res.text)` to get an object that is serializable to json. I should work on this, eh? If only xml could die in a fire.

**Comment by rtluckie** [comment]
@theherk agreed.  this is a pain point.  seria should handle FLOs as well as strings.  Make it so.
